nussl/separation/base/deep_mixin.py

```diff
@@ -27,20 +27,18 @@ def load_model(self, model_path, device='cpu'):
             metadata (dict): metadata associated with model, used for making
             the input data into the model.
         """
-        model_dict = torch.load(model_path, map_location='cpu')
-        model = SeparationModel(model_dict['config'])
-        model.load_state_dict(model_dict['state_dict'])
+        metadata = torch.load(model_path, map_location='cpu')
+        model = SeparationModel(metadata['config'])
```

Comment:
Hmm, not sure about the variable names here. `metadata` is the model? Can you add a description of `metadata` somewhere? I believe there's a section in the tutorial called "What's in a model".

Reply:
Still messing around with this a bit, it might change still.